templates/testpmd/scripts/change-nic-driver.sh

```diff
@@ -32,7 +40,7 @@ modprobe vfio-pci
 
 # Enable unsafe IOMMU.
 echo 1 > /sys/module/vfio/parameters/enable_unsafe_noiommu_mode
-conf
+echo "options vfio enable_unsafe_noiommu_mode=1" > /etc/modprobe.d/vfio-noiommu.conf
```

Comment:
I think this change is not related to this PR, also I saw the same change on https://github.com/RHsyseng/cnv-supplemental-templates/pull/7/files#diff-9a9eb2317c6991aa55853383742de3b38bcdd554c63b0aa63ff9669963146771R35